vault-tool/src/Network/VaultTool/Transit.hs

```diff
@@ -61,6 +63,10 @@ newtype CipherText = CipherText {getCipherText :: Base64}
     deriving (Eq, Ord, Read, Show, Generic)
     deriving newtype (A.FromJSON, A.ToJSON)
 
+
```

Comment:
nit: Would you mind running a formatter on this? @LaurentRDC 

Idk what we were using as a formatter before...